### PR TITLE
[bitnami/common] update imagePullSecrets to handle map and list format

### PR DESCRIPTION
**Pull request by greenday9**
[bitnami/common] update imagePullSecrets to handle map and list format

### Description of the change

This change allows imagePullSecrets to be either in list or map format.

```yaml
global:
  imagePullSecrets:
    - myRegistryKeySecretName
```

or

```yaml
global:
  imagePullSecrets:
    - name: myRegistryKeySecretName
```

### Benefits

Other charts use the map format and even helm create uses this format when generating a skeleton.  Having the flexibility to support both formats allows users to create umbrellas combining other charts with bitnami charts even if the other charts support map rendering of imagePullSecrets.

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/common/Chart.yaml [hunk 1]
```diff
@@ -23,4 +23,4 @@ name: common
 sources:
   - https://github.com/bitnami/charts
 type: library
-version: 2.13.0
+version: 2.13.1
```
Comment (migruiz4) [inline bitnami/common/Chart.yaml:26]: Could you please bump a new version? 

Version 2.13.1 has already been released: https://github.com/bitnami/charts/blob/main/bitnami/common/Chart.yaml

**Comment by migruiz4** [comment]
Thank you very much for your contribution! Changes look good to me, but a new version bump is required before I can approve it

File: bitnami/common/Chart.yaml (hunk 1 above)
Comment (jouve) [reply bitnami/common/Chart.yaml:26]:
```suggestion
version: 2.14.0
```

> Given a version number MAJOR.MINOR.PATCH, increment the:
>
>    MAJOR version when you make incompatible API changes
>    __MINOR version when you add functionality in a backward compatible manner__
>    PATCH version when you make backward compatible bug fixes

Comment (migruiz4) [reply bitnami/common/Chart.yaml:26]: @jouve This is not really a new feature such as adding a new helper, but a patch for an existing one

Comment (greenday9) [reply bitnami/common/Chart.yaml]: Thanks @migruiz4 , @jouve .  I've updated to 2.13.2.

**Review by migruiz4 — APPROVED** [review]

**Pull request merged**